api.web.yaml

```diff
@@ -251,6 +263,33 @@ components:
           type: string
           description: Time in RFC3339 format at which the the certificate was issued
 
+    TrainingRecord:
+      type: object
+      required:
+        - kind
+        - is_valid
+      properties:
+        kind:
+          type: string
+          description: The type of training (e.g., "nhsd")
```

Comment:
this should probably be an enum rather than a bare string (and referenced in both `ProfileTrainingUpdate` and here), but very happy to do that in a followup